lib/cryptography.js

```diff
@@ -14,7 +14,8 @@ const log = require('./utils/log')
 module.exports.decrypt = (options) => {
   try {
     const secret = options.secret || 'mySecret'
-    const inputFile = options.file || '.env.enc'
+    const inputFile = options.inputFile || '.env.enc'
```

Comment:
This would break https://github.com/kunalpanchal/secure-env/blob/master/lib/index.js#L18
Can we also update it there to send it as `inputFile` instead of `file`